### DISCUSSION TITLE
Wheelpicker selected border color

### DISCUSSION THREAD

**Discussion by batuhansahan** [wix/react-native-ui-lib#1878] [Q&A] [answered]
<img width="225" alt="Screen Shot 2022-03-03 at 23 20 24" src="https://user-images.githubusercontent.com/26772801/156646468-1707c38c-a0ee-4360-bfce-2ce079b6a1b8.png">

How to change these indicator colors.

**Comment by lidord-wix** [comment, marked as answer]
Featured added in this PR - https://github.com/wix/react-native-ui-lib/pull/1906
